OneCommunityGlobal/HighestGoodNetworkApp
# Nathan fix route permissions

# Description
This PR adds more permissions to have access to pages where they are relevant.

## Main changes explained:
- Remove `seeProjectManagement` and `seeProjectManagementTab` outdated permissions
- Updated Header to check for more permissions for access to Other Links
- Updated routePermissions.js with additional relevant permissions

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. login as an Owner user
5. go to Other Links -> Permissions Management -> Manage User Permissions
6. give a volunteer account one permission at a time for each permission
7. login with your volunteer (you will have to log out and back to update permissions if you were logged in in another tab/window)
8. go to the dashboard
9. Check that the links in Header -> Other Links/Reports work according to the Description column in [The Permissions Spreadsheet](https://docs.google.com/spreadsheets/d/1TfiJY9OLDZuyP2UgF0C1wI9tOSLKbS9MiJFH6n1Dsao/edit#gid=0). For example, users with `getReports` should have access to Reports->Reports in the Header.

**Jae's Note: I want to get this merged ASAP, so DM Nathan if any of this isn't clear.** 

## Notes
1. `getReports`: Reports->Reports
2. `getWeeklySummaries`: Reports->Weekly Summaries Reports
3. `getUserProfiles`, `postUserProfile`, `deleteUserProfile`, `changeUserStatus` : Other Links-> User Management
4. `seeBadges`, `createBadges`, `updateBadges`, `deleteBadges`: Other Links-> Badge Management
5. `postProject`, `deleteProject`, `putProject`, `getProjectMembers`, `assignProjectToUsers`, `postWbs`, `deleteWbs`, `postTask`, `updateTask`, `deleteTask`: Other Links -> Projects
6. `postTeam`, `putTeam`, `deleteTeam`, `assignTeamToUsers`: Other Links -> Teams
7. `postRole`, `deleteRole`, `putRole`, `putUserProfilePermissions`: Other Links -> Permissions Management
